api/src/pcapi/flask_app.py

```diff
@@ -88,7 +88,7 @@ def log_request_details(response: flask.wrappers.Response) -> flask.wrappers.Res
         "queryParams": request.query_string.decode(request.url_charset, errors="backslashreplace"),
         "size": response.headers.get("Content-Length", type=int),
         "deviceId": request.headers.get("device-id"),
-        "sourceIp": request.headers.get("X-Forwarded-For"),
+        "sourceIp": request.headers.get("X-Forwarded-For").split(",")[0],  # ignore the proxy ips
```

Comment:
Et s'il n'y a pas d'en-tête `X-Forwarded-For`, le `get()` renvoie-t-il `None` ? crash ?

Reply:
Le get semble renvoyer None

Reply:
Du coup crash sur `None.split()`

Reply:
Je m'en doutais :)